melos.yaml

```diff
@@ -131,6 +131,18 @@ scripts:
     melos exec -- \
       flutter analyze --no-fatal-infos --no-fatal-warnings &>/dev/null || true
 
+  pod:repo-update:
```

Comment:
I added these just to make running the example apps after a version update a bit less painful. There may be a better way to do this, but this is pretty much what I have been doing (just manually) after updating the version.

The issue is that after updating the version, if you run any example app you will likely see [this issue](https://docs.amplify.aws/lib/troubleshooting/upgrading/q/platform/flutter/).

Our clean script deletes podfile.lock files, so most/some of the time running a `melos clean && melos bootstrap` fixes it. But, that is still a pain, and will not resolve it if the new version isn't in your local pod spec.

Running `melos run pod:update` _should_ resolve it all scenarios and isn't as annoying as having to clean/bootstrap.

Reply:
thanks, I've noticed the same thing.